tests/foreman/api/test_docker.py

```diff
@@ -1346,10 +1343,25 @@ def test_positive_create_with_external_registry(self):
         @Assert: The docker container is created and the image is pulled from
         the external registry
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        repo_name = 'rhel'
+        registry = entities.Registry(url=DOCKER_0_EXTERNAL_REGISTRY).create()
+        try:
+            compute_resource = entities.DockerComputeResource(
+                organization=[self.org],
+                url=settings.docker.get_unix_socket_url(),
+            ).create()
+            container = entities.DockerRegistryContainer(
+                compute_resource=compute_resource,
+                organization=[self.org],
+                registry=registry,
+                repository_name='rhel',
```

Comment:
yeah, just mentioned and wanted to quickly fix it while no one noticed... wasted :smile: